tsconfig.json

```diff
@@ -3,11 +3,9 @@
     "module": "commonjs",
     "lib": [
       "dom",
-      "es5",
-      "es2015.collection",
-      "es2015.iterable"
+      "es2016"
```

Comment:
`es2016`? Is there something from `es2016` we're using or should this just be `es2015`?